### PR TITLE
[zh] Link to new API reference page for APIService

### PR DESCRIPTION
**Pull request by mengjiao-liu**
[zh] sync content/en/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md file
[zh] Link to new API reference page for APIService
en PR: https://github.com/kubernetes/website/pull/29228

### CODE REVIEW

File: content/zh/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md [hunk 1]
```diff
@@ -78,12 +78,14 @@ If your extension API server cannot achieve that latency requirement, consider m
...
+
+Alternatively: learn how to [extend the Kubernetes API using Custom Resource Definitions](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/).
 -->
 * 阅读[配置聚合层](/zh/docs/tasks/extend-kubernetes/configure-aggregation-layer/) 文档，
   了解如何在自己的环境中启用聚合器。
 * 接下来，了解[安装扩展 API 服务器](/zh/docs/tasks/extend-kubernetes/setup-extension-api-server/)，
   开始使用聚合层。
-* 也可以学习怎样[使用自定义资源定义扩展 Kubernetes API](/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)。
-* 阅读 [APIService](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#apiservice-v1-apiregistration-k8s-io) 的规范
+* 从 API 参考中阅读关于[APIService](/docs/reference/kubernetes-api/cluster-resources/api-service-v1/) 的信息。
```
Comment (xichengliudui) [inline content/zh/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md:89]:
```suggestion
* 从 API 参考中阅读关于 [APIService](/docs/reference/kubernetes-api/cluster-resources/api-service-v1/) 的信息。
```

Comment (mengjiao-liu) [reply content/zh/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md]: Thanks for catch!  Updated.

File: content/zh/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md [hunk 2]
```diff
@@ -78,12 +78,14 @@ If your extension API server cannot achieve that latency requirement, consider m
...
+
+Alternatively: learn how to [extend the Kubernetes API using Custom Resource Definitions](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/).
 -->
 * 阅读[配置聚合层](/zh/docs/tasks/extend-kubernetes/configure-aggregation-layer/) 文档，
   了解如何在自己的环境中启用聚合器。
 * 接下来，了解[安装扩展 API 服务器](/zh/docs/tasks/extend-kubernetes/setup-extension-api-server/)，
   开始使用聚合层。
-* 也可以学习怎样[使用自定义资源定义扩展 Kubernetes API](/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)。
-* 阅读 [APIService](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#apiservice-v1-apiregistration-k8s-io) 的规范
+* 从 API 参考中阅读关于 [APIService](/docs/reference/kubernetes-api/cluster-resources/api-service-v1/) 的信息。
```
Comment (ydFu) [inline content/zh/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md:89]:
```suggestion
* 从 API 参考资料中研究关于 [APIService](/docs/reference/kubernetes-api/cluster-resources/api-service-v1/) 的内容。
```

File: content/zh/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md [hunk 3]
```diff
@@ -78,12 +78,14 @@ If your extension API server cannot achieve that latency requirement, consider m
...
+
+或者，可以学习怎样[使用自定义资源定义扩展 Kubernetes API](/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)。
```
Comment (ydFu) [inline content/zh/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md:91]:
```suggestion
或者，学习如何[使用自定义资源定义扩展 Kubernetes API](/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)。
```

File: content/zh/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md (hunk 2 above)
Comment (mengjiao-liu) [reply content/zh/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md]: Thanks for your suggestion. Updated.

Comment (mengjiao-liu) [reply content/zh/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md]: 我参考了其他中文翻译页面对 `API reference` 的翻译，都是翻译为 `API 参考`,是否遵循前例比较好？ 
/cc @tengqm

**Pull request merged**